src/utils/routeCreators.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const createHomeRoute = (userType) => `/${userType}/home`;
```

Comment:
Are you going to create all users like this?
Maybe create some route, which will automatically get the type from the store?

Reply:
How do you imagine it?